src/_webp.c

```diff
@@ -15,10 +13,8 @@
  * very early versions had some significant differences, so we require later
  * versions, before enabling animation support.
  */
-#if WEBP_MUX_ABI_VERSION >= 0x0104 && WEBP_DEMUX_ABI_VERSION >= 0x0105
```

Comment:
This ABI versions actually was never released, v0.5.0 have 0x0106 and 0x0106

Reply:
IIRC, some of that was for webp as shipped in debian derived distributions, now long deprecated.